src/amo/sagas/reviews.js

```diff
@@ -200,7 +209,16 @@ function* manageAddonReview(
   const { body, errorHandlerId, rating } = action.payload;
   const errorHandler = createErrorHandler(errorHandlerId);
 
+  const savingRating = !!rating;
+  const savingReview = !!body;
```

Comment:
With the new set up it always be one or the other (even on the initial create), is that right?

Reply:
It could be both. They are independent of each other.